tests/unit/backend/corpora/common/test_authorizer.py

```diff
@@ -1,23 +1,27 @@
+import json
 import os
 import unittest
 
 import requests
 from chalice import UnauthorizedError
 
 from backend.corpora.common.authorizer import assert_authorized
+from backend.corpora.common.utils.aws_secret import AwsSecret
 
 if not os.getenv("DEPLOYMENT_STAGE"):  # noqa
     os.environ["DEPLOYMENT_STAGE"] = "test"  # noqa
 
+secret_name = "corpora/cicd/dev/auth0-secret"
+# using the same secret for all none production stages.
 
-@unittest.skipIf(os.getenv("DEPLOYMENT_STAGE") != "test", "DEPLOYMENT_STAGE not 'test'")
+
+@unittest.skipIf(os.getenv("DEPLOYMENT_STAGE", "test") == "test", "DEPLOYMENT_STAGE is 'test'")
```

Comment:
I think the second argument of `skipIf` is a justification on why it. was skipped in which case is should be that DEPLOYMENT_STAGE was _not_ set to `test`.

Reply:
I was going to limit the frequency of this test to only run on pushed to master until we find a better way of managing the generated token. This limits the test to only run when deployment is Dev above.